tests/ert_tests/ert3/engine/integration/test_engine.py

```diff
@@ -481,3 +491,61 @@ def test_sensitivity_run_and_export(
     assert_sensitivity_oat_export(
         export_data, sensitivity_ensemble, stages_config, gaussian_parameters_config
     )
+
+
+@pytest.mark.requires_ert_storage
+def test_partial_sensitivity_run_and_export(
+    workspace,
+    partial_sensitivity_ensemble,
+    double_stages_config,
+    sensitivity_experiment_config,
+    oat_compatible_record_file,
+    gaussian_parameters_config,
+):
+    (workspace / ert3.workspace.EXPERIMENTS_BASE / "partial_sensitivity").ensure(
+        dir=True
+    )
+    ert3.engine.load_record(workspace, "other_coefficients", oat_compatible_record_file)
+    ert3.engine.run(
+        partial_sensitivity_ensemble,
+        double_stages_config,
+        sensitivity_experiment_config,
+        gaussian_parameters_config,
+        workspace,
+        "partial_sensitivity",
+    )
+    ert3.engine.export(workspace, "partial_sensitivity")
+
+    export_data = _load_export_data(workspace, "partial_sensitivity")
+    assert_sensitivity_oat_export(
+        export_data,
+        partial_sensitivity_ensemble,
+        double_stages_config,
+        gaussian_parameters_config,
+    )
+
+
+@pytest.mark.requires_ert_storage
+def test_incompatible_partial_sensitivity_run(
+    workspace,
+    partial_sensitivity_ensemble,
+    double_stages_config,
+    sensitivity_experiment_config,
+    oat_incompatible_record_file,
+    gaussian_parameters_config,
+):
+    (workspace / ert3.workspace.EXPERIMENTS_BASE / "partial_sensitivity").ensure(
+        dir=True
+    )
+    ert3.engine.load_record(
+        workspace, "other_coefficients", oat_incompatible_record_file
+    )
+    with pytest.raises(AssertionError):
```

Comment:
Please `match` the exception message. Also, since this is an exception that could easily be produced by the user we should not make it an `AssertionError`. Second, we should get this into the console without a stack trace. This is handled other places by using `ERTError`. But, the second part we can leave in an issue for later work...

Reply:
Any tips on how to match a string that is continued over multiple lines to fix line too long?

Reply:
You can match against a substring. Or you can create a string containing the message up front:

```python
msg = (
    "This is one lone string "
    "with multiple lines:"
    "\n\n\n\n"
)
```